README.md

```diff
@@ -20,7 +20,7 @@ You will need:
 Install dependencies:
 
 ```
-$ yarn install
+$ yarn install --production=false
```

Comment:
It isn't necessary to specify this flag for development. `devDependencies` will always be installed.